index.html

```diff
@@ -2,18 +2,127 @@
 
 <html lang="en">
 <head>
+  <link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet'>
   <meta charset="utf-8">
 
   <title>My Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
 
+  <style>
+    body {
+      font-family: Oswald; font-size: 22px;
+    }
```

Comment:
Nice job using Google fonts. This is a good use case for a Less variable. Ex:
`@body-font: Oswald, arial, sans-serif;`